### PR TITLE
11 apr stg

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: docs/whats-new/production-release-notes.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Frontend version: 1.0.0-20240306.1552 | Backend version: 1.95.14-20240311.1204
 
 | Type | Change| Description|
  ------ | ------- | ----------- |
-| **Feature** | Expose SAML features | Tenants can now manage SAML apps on the portal. Refer to [SAML Apps](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
+| **Feature** | Expose SAML features | Tenants can now manage SAML apps on the portal. Refer to [SAML Apps](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
```
Comment (yqyeoh) [inline docs/whats-new/production-release-notes.md]: sorry! I just realised recently saml apps is only viewable, not manageable, can we change to 'can now view' (change in both stg & prod release notes)

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+## April 2024
+
+**11 April 2024**
+
+Frontend version: 1.0.0-20240328.0943 | Backend version: 1.97.2-20240405.1708
+
+
+| Type | Change | Description |
+| ---- | ------ | ----------- |
+| **Feature** | SAML certificates expiry notifications | The `notify-expiring-saml-certs` job has been enhanced to email tenant admins about SAML certificate expirations. Notifications will exclude certificates for applications not aligned with TechPass standards. |
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]: Dun need to mention the job name (it's unknown to the tenants) & remove the 'Notifications will exclude...' sentence

I think sth like 'Tenant admins will receive email notifications when an active SAML app certificate is nearing expiration' will do.

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+| **Fix** | Fixed the app role assignments UI issue | Resolved an issue where the `next link` in the app role assignments section was not functioning correctly on the UI, enhancing the user experience and navigability within the platform. |
+
+
+Frontend version: 1.0.0-20240321.1254 | Backend version: 1.95.26-20240325.1331
+
+| Type | Change | Description |
+|------|--------|-------------|
+| **Feature** | Expose app roles management on TechPass portal | Tenants can now manage app roles (excluding app roles used by TechBiz) on the portal under the *edit application* page. |
+| **Change** | Autonomous app role assignments for TechBiz-integrated applications | Previously available only via Automation API, tenants can now autonomously manage app role assignments for TechBiz-integrated applications (excluding TechBiz group assignments) directly through the TechPass portal. |
+
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md:17]: i think u intend to delete these

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: I don't think we need to mention the job name.

File: docs/whats-new/production-release-notes.md (hunk 1 above)
Comment (JLeeChan) [reply docs/whats-new/production-release-notes.md]: should we call out that cannot manage?

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: i removed this alr
 @JLeeChan

File: docs/whats-new/staging-release-notes.md (hunk 3 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md:17]: deleted

File: docs/whats-new/staging-release-notes.md [hunk 4]
```diff
@@ -10,7 +31,6 @@ Frontend version: 1.0.0-20240321.1254 | Backend version: 1.95.26-20240325.1331
 | Type | Change | Description |
 |------|--------|-------------|
 | **Feature** | Expose app roles management on TechPass portal | Tenants can now manage app roles (excluding app roles used by TechBiz) on the portal under the *edit application* page. |
-| **Change** | Autonomous app role assignments for TechBiz-integrated applications | Previously available only via Automation API, tenants can now autonomously manage app role assignments for TechBiz-integrated applications (excluding TechBiz group assignments) directly through the TechPass portal. |
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md:13]: why delete this?

File: docs/whats-new/staging-release-notes.md (hunk 3 above)
Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md:17]: i mean why do we have line 17 to line 22?

File: docs/whats-new/production-release-notes.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ Frontend version: 1.0.0-20240306.1552 | Backend version: 1.95.14-20240311.1204
 
 | Type | Change| Description|
  ------ | ------- | ----------- |
-| **Feature** | Expose SAML features | Tenants can now manage SAML apps on the portal. Refer to [SAML Apps](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
+| **Feature** | Expose SAML features | Tenants can now view SAML apps on the portal. Refer to [SAML Apps](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
```
Comment (yqyeoh) [inline docs/whats-new/production-release-notes.md:25]: can we update the stg release note to change 'manage' to 'view' as well

**Review by yqyeoh — APPROVED** [review]

**Pull request merged**